apps/antalmanac/src/stores/AppStore.ts

```diff
@@ -49,25 +52,29 @@ class AppStore extends EventEmitter {
     }
 
     getCurrentScheduleIndex() {
-        return this.schedule.getCurrentScheduleIndex();
+        return this.schedules.getCurrentScheduleIndex();
```

Comment:
Just a complaint with the app store and not something you have to fix.
Why is there an abstraction over the schedules? It makes literally no sense. There's one app store in the app, and one schedule store in the app. If you want to subscribe to a particular data source, just pick the one you need.

But coercing the schedules into the schedule store just makes a really bloated class for no reason, with the guise that it's the single source of truth (it isn't and it doesn't make anything simpler).

Reply:
I'd imagine that's because schedule used to be a simple data structure that existed in AppStore, then it was extracted when it got too complicated.

Also, if you feel like talking OOP, it's about AppStore owning the schedule class, which is why OOP is not good

Reply:
Based oop hater 😎